test/parser/chord_sheet_parser.js

```diff
@@ -17,16 +17,16 @@ describe('ChordSheetParser', () => {
     expect(lines.length).toEqual(2);
 
     const line0Items = lines[0].items;
-    expect(line0Items[0]).toBeChordLyricsPair('', 'Let it ');
-    expect(line0Items[1]).toBeChordLyricsPair('Am', 'be, let it ');
-    expect(line0Items[2]).toBeChordLyricsPair('C/G', 'be, let it ');
-    expect(line0Items[3]).toBeChordLyricsPair('F', 'be, let it ');
+    expect(line0Items[0]).toBeChordLyricsPair('      ', 'Let it ');
+    expect(line0Items[1]).toBeChordLyricsPair('Am        ', 'be, let it ');
+    expect(line0Items[2]).toBeChordLyricsPair('C/G       ', 'be, let it ');
+    expect(line0Items[3]).toBeChordLyricsPair('F         ', 'be, let it ');
     expect(line0Items[4]).toBeChordLyricsPair('C', 'be');
 
     const line1Items = lines[1].items;
-    expect(line1Items[0]).toBeChordLyricsPair('C', 'Whisper words of ');
-    expect(line1Items[1]).toBeChordLyricsPair('G', 'wisdom, let it ');
-    expect(line1Items[2]).toBeChordLyricsPair('F', 'be');
+    expect(line1Items[0]).toBeChordLyricsPair('C               ', 'Whisper words of ');
+    expect(line1Items[1]).toBeChordLyricsPair('G             ', 'wisdom, let it ');
+    expect(line1Items[2]).toBeChordLyricsPair('F ', 'be');
     expect(line1Items[3]).toBeChordLyricsPair('C/E', '');
```

Comment:
How about adding the white space on the lyrics property like this?
```javascript
expect(line1Items[3]).toBeChordLyricsPair('C/E', '    '); // 4 spaces = 'C/E'.length + space(s) to the next chord
expect(line1Items[4]).toBeChordLyricsPair('Dm', '   ');
expect(line1Items[5]).toBeChordLyricsPair('C', ' ');
```

Reply:
I decided to parse everything as-is, and leave it up to the formatter how to format it.